pgatk/commands/ncbi_downloader.py

```
@@ -0,0 +1,36 @@
+import click
+
+from pgatk.clinvar.data_downloader import NcbiDataDownloader
+from pgatk.config.registry import load_config
+
+
+@click.command("ncbi-downloader", short_help="Download NCBI RefSeq and ClinVar reference files")
+@click.option("-c", "--config_file", help="Configuration YAML file (optional)")
+@click.option("-o", "--output-dir", required=True, help="Output directory for downloaded files")
+@click.option("--force", is_flag=True, default=False, help="Re-download files even if they exist")
+@click.pass_context
+def ncbi_downloader(ctx, config_file, output_dir, force):
```

Ruff (0.15.2)

[warning] Unused function argument: `ctx`

(ARG001)